app/build.gradle

```diff
@@ -97,6 +99,7 @@ def retrofitVersion = '2.6.1'
 def picassoVersion = '2.5.2'
 def assertjVersion = '1.2.0'
 def acraVersion = '5.4.0'
+def espressoVersion = '3.3.0-alpha03'
```

Comment:
In general I think it is wise to use only released versions and avoid alpha and beta, but for the testing framework I have not at all an issue with it.

Reply:
I can try with others, but that's the one copied from the documentation :)

Reply:
no need... we don't need to fear big incompatible changes from alpha03 to something released then.